store/iscn.ts

```diff
@@ -161,6 +163,25 @@ export default class ISCN extends VuexModule {
     return addIDToRecords(result);
   }
 
+  @Action
+  async queryISCNByField({
+    keyword,
+  }: { keyword: string }): Promise<ISCNRecordWithID[]> {
+    this.context.commit('clearRecords');
+    let result: ISCNRecord[] = ([] as ISCNRecord[])
+    try {
+      this.context.commit('setIsLoading', true)
+      const { data } = await axios.get(`${network.apiURL}/iscn/records?keyword=${encodeURIComponent(keyword)}`)
```

Comment:
I think it should be `/iscn/keywords?q=` for search all function (includes iscn id, owner, stakeholder, keywords). `keyword` will only search `contentMetadata.keywords`.

Reply:
yes, the requirement was to specifically query by `keyword`. I think `queryISCNByKeyword`(poor name btw) above should be using `q` for search-all, and this function is for specific fields
https://github.com/likecoin/app-like-co/pull/175 should replace the above function by `q` after we fix performance issue (I based this PR on #175 actually)